clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -941,6 +941,19 @@ struct CounterCoverageMappingBuilder
     return Counter::getCounter(CounterMap[S]);
   }
 
+  std::pair<Counter, Counter> getBranchCounterPair(const Stmt *S,
```

Comment:
it's not immediately obvious what the pair represents from this function name.

`getExecutionCountAnd...`?

or, make a lightweight POD struct to represent this

```
struct ExecCntAnd... { 
  Counter ExecCnt
  Counter ...
};
```

Reply:
I like a silly `std::pair` :) Let me think more.

Reply:
I think changing the name is sufficient for making me less confused